Source/WebCore/svg/SVGElement.cpp

```diff
@@ -243,7 +244,7 @@ void SVGElement::reportAttributeParsingError(SVGParsingError error, const Qualif
     if (error == NoError)
         return;
 
-    String errorString = "<" + tagName() + "> attribute " + name.toString() + "=\"" + value + "\"";
+    String errorString = "<" + tagName() + "> attribute " + name.toString() + "=" + JSON::Value::escapeString(value);
```

Comment:
You can add this static function to `JSON::Value` and it will make things compile:

```
String Value::escapeString(StringView string)
{
    StringBuilder result;
    escapeString(result, string);
    return result.toString();
}
```

Reply:
Sure, I will reopen and push an update.

Reply:
I tried but it didn't work and then I tried to look into Chromium implementation and trying to see if it works but I think it would be difficult. My local build environment is also not working.